src/main/java/ip/Task.java

```diff
@@ -0,0 +1,40 @@
+package ip;
+
+class Task {
+    protected String description;
+    protected boolean isDone;
+
+    Task(String description) {
+        this(description, false);
+    }
+
+    protected Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
```

Comment:
Perhaps you can simplify your constructors by marking `isDone` to be `false` by default.
```suggestion
    protected boolean isDone = false;

    Task(String description) {
        this.description = description;
    }
    
    protected Task(String description, boolean isDone) {
        this(description);
        this.isDone = isDone;
    }
```